src/editor/components/TagFilter.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { View, TouchableOpacity, StyleSheet, Text } from 'react-native';
```

Comment:
View isn't being used currently, can be removed

Reply:
Done!